src/gui/wizard/owncloudadvancedsetuppage.cpp

```diff
@@ -457,4 +535,30 @@ void OwncloudAdvancedSetupPage::setRadioChecked(QRadioButton *radio)
         _ui.rVirtualFileSync->setCheckable(false);
 }
 
+void OwncloudAdvancedSetupPage::setupSyncLogo()
+{
+    _ui.syncLogoLabel->setPixmap(Theme::hidpiFileName(":/client/theme/white/sync-arrows.png"));
+}
+
+void OwncloudAdvancedSetupPage::setupSyncModeLabel()
+{
+    auto sizePolicy = _ui.syncModeLabel->sizePolicy();
+    sizePolicy.setRetainSizeWhenHidden(true);
```

Comment:
Just wondering, wouldn't doing this at the page level be enough instead of doing it per widget?

Reply:
If I do this on the page the layout of the other pages changes too

Reply:
Well, isn't it what we want? Because we want all the pages to have the same size in the end to avoid resizes on step changes? So wouldn't that lead to not needing the `adjustWizardSize` in the wizard class?

Reply:
Sorry I wasn't specific enough. If I set the size policy on the `OwncloudAdvancedSetupPage` int the constructor then I get a incorrect layout on all the other pages. For example https://cloud.nextcloud.com/s/g5maTToEcwERY3t . I'm unsure what here is happening.

Reply:
Hm... looks like it's overconstrained somehow... Or that one of the components claims a too small size... would be a good target for investigating with Gammaray.

Reply:
@FlexW I agree with @er-vin. It would be nice to have it somewhere in a single place.

Reply:
I agree with you both. I try to fix that.

Reply:
After some more investigation I'm still unable to solve that. Are you sure that it would be enough to set the size policy on the page level because from my observations in gammaray the size policy got not propagated to the child widgets and I couldn't find any documentation that states something else. However, I simplified the setting of the size policy a little bit so that it makes less noise.